fit/core-reference/src/test/java/org/apache/syncope/fit/core/UserIssuesITCase.java

```diff
@@ -1820,4 +1826,58 @@ void issueSYNCOPE1818() {
             jdbcTemplate.update("DELETE FROM TESTPULL WHERE USERNAME = 'rossini'");
         }
     }
+
+    @Test
+    void issueSYNCOPE1853() {
+        UserTO bellini = USER_SERVICE.read("bellini");
```

Comment:
Why the need to read `bellini` and `vivaldi` if the following code is just using their keys?

Reply:
For readability, reverting to keys.